scripts/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then 
+    BUILD_TYPE=Release
+else 
+    BUILD_TYPE="$1"
+fi
+
+PWD=$(pwd)
```

Comment:
ah ok. To restore the pwd, I usually just start a subshell:
```
(
do you stuff
)
```
bash will automatically take you back.